default/roles/nginx.json

```diff
@@ -10,6 +10,7 @@
     "run_list": [
         "recipe[chef-solo-search]",
         "recipe[apt]",
+        "recipe[nginx-hardening::upgrades]",
```

Comment:
sure, that would be a more direct way. my line of thinking was to make it more visual and extendable from the beginning. but that can be delayed until we really need more than just one thing